externs/shaka/player.js

```diff
@@ -1095,7 +1102,6 @@ shaka.extern.MssManifestConfiguration;
  * @property {boolean} raiseFatalErrorOnManifestUpdateRequestFailure
  *   If true, manifest update request failures will cause a fatal error.
  *   Defaults to <code>false</code> if not provided.
- *
```

Comment:
revert it

Reply:
Done

Reply:
I think defaulting to `false` more accurately describes the current behavior. We have manifests that do not have consistent streams across periods, so if it defaults to `true` we would have to opt-out when upgrading to this version.